### PR TITLE
extended some exercise from a Ubuntu VM

### PR DESCRIPTION
**Pull request by tuxerrante**
Hi,
it is still a work in progress, for now I had
- removed the pdf since we could receive complains from the course authors. The idea is to integrate all the exercises in the txt/md files anyway
- Created new exercises for mariaDB, LXC, rsyslog and Apache
- Rewritten the Readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,91 @@
...
+        Configure advanced PAM
+
+    Networking 15%
+        Monitor, tune and troubleshoot network performance
+        Configure network traffic tunneling
+        Configure a system to perform Network Address Translation
+        Dynamically route IP traffic
+        Implement advanced packet filtering
+
+    Service Configuration17%
```
Comment (ghumman) [inline README.md]: Here's a typo.
`Service Configuration 17%`

File: README.md [hunk 2]
```diff
@@ -1,11 +1,91 @@
...
+## Requirements
+https://training.linuxfoundation.org/certification/linux-foundation-certified-engineer-lfce/
+
+    Essential Commands 5%
+        Use version control tools
+        Manipulate file content programmatically
+        Run commands on many systems simultaneously
+        Install Linux Distribution
+
+    Operation of Running Systems 18 %
```
Comment (ghumman) [inline README.md:34]: Not consistent.
`Operation of Running Systems 18%`

**Review by ghumman — COMMENTED** [review]
Looking great, requesting minor changes.

File: README.md (hunk 2 above)
Comment (tuxerrante) [reply README.md:34]: What do you mean? 
That the official docs https://training.linuxfoundation.org/certification/linux-foundation-certified-engineer-lfce/

Comment (realophobia) [reply README.md:34]: There was an extra space between 18 and % sign, but not a big deal. Merging now.

**Comment by ghumman** [comment]
Hi,

Thanks for the PR. I think your commits are missing email and name, maybe
that's why you're not showing up as a contributor in the repo after the PR
is merged. Can you please update the email and name using following
commands and push a PR so that your name can show up as a contributor. You
can resend the same PR with updated email and name or make some other minor
changes and push the code.

git config --global user.name "FIRST_NAME LAST_NAME"
git config --global user.email ***@***.***"

Thanks.

**Comment by tuxerrante** [comment]
Thanks for the advice, I just switched laptop and I have some missing configuration :)

**Comment by ghumman** [comment]
I rebased your commits and fixed name and email address for your commits
and I can see you as a contributor now.

Thanks.
Ahmed.

**Pull request merged**